### PR TITLE
#161747559 Fetch all articles route

### PR DESCRIPTION
**Pull request by KvNGCzA**
#### What does this PR do?
Have `/api/v1/articles` route working

#### Description of Task to be completed?
- create `/api/v1/articles` route
- create fetch articles controller
- create tests for fetch articles route
- correct error thrown by twitter test

#### How should this be manually tested?
- `npm start` to start your server
- use your browser or postman to hit the`localhost:{PORT}/api/v1/articles` route

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- [#161747559](https://www.pivotaltracker.com/story/show/161747559)

#### Screenshots (if appropriate)
screen shot 2018-11-10 at 5 17 41 pm

#### Questions:
- None

### CODE REVIEW

**Comment by mbilesanmi** [comment]
I believe you should load some of the user-info also for each article.

**Comment by KvNGCzA** [comment]
> I believe you should load some of the user-info also for each article.

@mbilesanmi changes implemented

File: server/controllers/ArticleController.js [hunk 1]
```diff
@@ -75,6 +76,34 @@ class ArticleController {
...
+      })
+      .then((result) => {
+        res.status(200).json({
+          Articles: result.rows
+        });
+      })
+      .catch(err => res.status(500)
+        .json({
+          errors: {
+            message: [err.message]
```
Comment (sulenchy) [inline server/controllers/ArticleController.js]: I think err.message might not be descriptive enough. I suggest you replace that with a descriptive statement such `Something went wrong. Try again or contact the administrator`. Aside from that, every other thing looks good to me.

**Comment by augustineezinwa** [comment]
Just as @mbilesanmi said, you could populate each of the User objects in the Articles array. Do well to change the key `User` to `author` refer to this [https://github.com/andela/valinor-ah-backend#multiple-articles](url)

**Comment by KvNGCzA** [comment]
> Just as @mbilesanmi said, you could populate each of the User objects in the Articles array. Do well to change the key `User` to `author` refer to this [https://github.com/andela/valinor-ah-backend#multiple-articles](url)

@augustineezinwa done

File: server/controllers/ArticleController.js (hunk 1 above)
Comment (sulenchy) [reply server/controllers/ArticleController.js]: Okay

**Comment by mrSerious** [comment]
This is good work with this @KvNGCzA. May I suggest you include a screenshot from Postman in you PR description? I think that it will make the PR look even better.

**Comment by KvNGCzA** [comment]
> Good work with this @KvNGCzA. May I suggest you include a screenshot from Postman in you PR description? I think that it will make the PR look even better.

@mrSerious Done. Thanks

File: server/migrations/20181029191911-create-user.js [hunk 2]
```diff
@@ -33,15 +33,15 @@ export default {
     },
     facebookId: {
       allowNull: true,
-      type: Sequelize.STRING,
+      type: Sequelize.BIGINT,
```
Comment (mbilesanmi) [inline server/migrations/20181029191911-create-user.js]: Are these ids integer?

Comment (KvNGCzA) [reply server/migrations/20181029191911-create-user.js]: @mbilesanmi yes they are

**Pull request merged**